repose-aggregator/components/filters/valkyrie-authorization-filter/src/main/resources/META-INF/schema/config/bindings.xjb

```diff
@@ -27,6 +27,6 @@
           schemaLocation="valkyrie-authorization.xsd">
 
     <schemaBindings>
-        <package name="org.openrepose.filters.valkyrieauthorization.config"/>
+        <package name="org.openrepose.filters.scripting.config"/>
```

Comment:
I must have done a replace at some point. The scripting filter started as a copy/paste of the valkyrie filter, so I must have done a replace to update all of that jank, but ended up replacing that jank where it shouldn't have been replaced.